tests/test_sqs/test_sqs.py

```diff
@@ -777,6 +777,38 @@ def test_send_receive_message_with_attributes_with_labels():
     )
 
 
+@mock_sqs
+def test_change_message_visibility_than_permitted():
```

Comment:
We'll need to skip this test in ServerMode (similar to this test https://github.com/spulec/moto/blob/master/tests/test_sqs/test_sqs.py#L1820)

Reply:
Done